test.js

```diff
@@ -86,6 +92,18 @@ describe('gitlab urls', function () {
     assert.deepEqual(['user', 'test1', ''], parsed)
   })
 
+  it('parses https non-itlab url', function () {
+    var url = 'https://git.mycompany.com/user/test1.git'
+    var parsed = parse(url)
+    assert.deepEqual(['user', 'test1', ''], parsed)
+  })
+
+  it('parses https non-itlab url', function () {
+    var url = 'https://git.mycompany.local/user/test1.git'
```

Comment:
Can this either be combined with the above test case, or have the description updated to reflect that you're testing a `local` TLD of a GitLab deployment?